service/models.py

```diff
@@ -145,12 +178,30 @@ def init_db(cls, app):
         app.app_context().push()
         db.create_all()  # make our sqlalchemy tables
 
+    @classmethod
+    @retry(
+        HTTPError,
+        delay=RETRY_DELAY,
+        backoff=RETRY_BACKOFF,
+        tries=RETRY_COUNT,
+        logger=logger,
+    )
+
     @classmethod
```

Comment:
this method still has a `@classmethod` decorator (in addition to the one being added above), please remove this one with this change